client/src/app/site/base/base-user-import.service.ts

```diff
@@ -16,6 +18,9 @@ export abstract class BaseUserImportService extends BaseImportService<User> {
         if (header === `is_active` || header === `is_physical_person`) {
             return this.toBoolean(value);
         }
+        if (header === `gender`) {
+            return this.getGenderInEnglish(value);
```

Comment:
Is this analogous to the group name translation matching? I can't seem to find it.

Reply:
It probably is, I wrote this function because I didn't find the place where we do that, but I guess I'll look again.

Reply:
Regrettably I still can't find it

Reply:
ok